README.md

```diff
@@ -64,3 +64,14 @@ API](https://docs.newrelic.com/docs/introduction-new-relic-metric-api) and [New
 Relic Trace API Requirements and
 Limits](https://docs.newrelic.com/docs/apm/distributed-tracing/trace-api/trace-api-general-requirements-limits)
 on the specifics of the rate limits.
+
```

Comment:
I think if we moved this section up to be after "Using the exporter" it would flow a bit better.